python/pyspark/cloudpickle.py

```diff
@@ -42,20 +42,26 @@
 """
 from __future__ import print_function
 
-import dis
-from functools import partial
-import imp
 import io
-import itertools
-import logging
+import dis
```

Comment:
Not a big deal but did these imports need to shuffle? I'm not clear what was added or removed

Reply:
This is a copy of cloudpickle 0.6.1. I think we should match it and avoid some divergence. It needed a lot of effort to match it before.

Reply:
Ah OK I understand.

Reply:
> Not a big deal but did these imports need to shuffle? I'm not clear what was added or removed

It's an exact copy of cloudpickle 0.6.1 code. So the import order was not shuffled by me on purpose.